Dockerfile

```diff
@@ -257,6 +257,11 @@ COPY --from=gcr.io/projectsigstore/cosign:v1.3.1@sha256:3cd9b3a866579dc2e0cf2fde
 COPY ./Dockerfile.d/test-integration-etc_containerd-stargz-grpc_config.toml /etc/containerd-stargz-grpc/config.toml
 COPY ./Dockerfile.d/test-integration-ipfs-offline.service /usr/local/lib/systemd/system/
 COPY ./Dockerfile.d/test-integration-buildkit-nerdctl-test.service /usr/local/lib/systemd/system/
+COPY ./Dockerfile.d/SHA256SUMS.d/tinit-static-v0.19.0 /SHA256SUMS.d/tinit-static-v0.19.0
```

Comment:
`/Dockerfile.d/SHA256SUMS.d/` already copied to the image. https://github.com/containerd/nerdctl/blob/master/Dockerfile#L109

Reply:
https://github.com/containerd/nerdctl/blob/master/Dockerfile#L210-L211

But in the multi staging build, we don't copy the sum file to the test stage

Reply:
Why do you install tini only to the test stage